### DISCUSSION TITLE
How to analyse trades?

### DISCUSSION THREAD

**Discussion by shikharvaish28** [ssantoshp/Empyrial#116] [Q&A] [answered]
Hey @ssantoshp , I just stumbled on this library and it seems great. I have an algotrader and I want to analyse its trades. Can I use your library to analyse its trades.

Here is the screenshot of the order log. The stocks in it changes everyday depending on my strategy. If analysis of it is possible then I would love to use it in my analytics layer.

<img width="1375" alt="Screenshot 2023-10-15 at 1 37 17 AM" src="https://github.com/ssantoshp/Empyrial/assets/22795943/2cac6429-90aa-4bf8-b448-f62aa0628e03">

**Comment by santoshlite** [comment, marked as answer]
Hmm, that would not be possible right now because Empyrial focuses more on quickly evaluating investment ideas rather than analyzing trade data. That's something I'll keep in the corner of my mind tho because it's a good idea!

**Reply by shikharvaish28** [reply]
Thanks! Watching this repo for whenever you make this update. Kudos to your effort :)

**Reply by BradKML** [reply]
Would like to know more about trade analysis since there are libraries that does that but not necessarily faster + they might not have optimizer built-in for technical indicators or ML predictions.
